Samples/iOS-Swift/iOS-Swift/Base.lproj/Main.storyboard

```diff
@@ -183,23 +183,23 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="ntC-Ea-bhK">
-                                <rect key="frame" x="0.0" y="64" width="320" height="400.5"/>
+                                <rect key="frame" x="0.0" y="64" width="320" height="432.5"/>
```

Comment:
Not for this PR, but I would like to remove any storyboard

Reply:
agreed 👍🏻